nautobot/extras/views.py

```diff
@@ -970,10 +973,16 @@ def get(self, request, class_path=None, slug=None):
 
     def post(self, request, class_path=None, slug=None):
         job_model = self._get_job_model_or_404(class_path, slug)
+        template_name = "extras/job.html"
+        try:
+            job_class = job_model.job_class()
+            job_form = job_class.as_form(request.POST, request.FILES) if job_model.job_class is not None else None
+            if hasattr(job_class, "template_name"):
+                template_name = job_class.template_name
+        except RuntimeError as err:
```

Comment:
I'm not exactly sure, i did not look into it. This was the [in-place](https://github.com/nautobot/nautobot/blob/develop/nautobot/extras/views.py#L953-L957) exception handling that was used in the codebase for the existing `get` method when trying to access the job class. I can remove it if its not required

Reply:
I missed that this was patterned after the existing code above 🤦 . Fine to leave as-is!

Reply:
I removed this because the error condition this was handling is already being handled below and in fact changes the existing behavior:

https://github.com/nautobot/nautobot/blob/next/nautobot/extras/views.py#L999-L1000

When jobs were no longer installed it redirected to the list view and failed the tests previously for that. Instead of changing that behavior we have slightly duplicate check for the class existing.

The `get` and `post` could probably be made more succinct with their checks, error messaging, and return paths but I believe they are fine as-is at the moment.